History.md

```diff
@@ -1,42 +1,66 @@
-# grunt-banana-checker Release History
+v0.7.0 / 2019-01-08
+==================
+
+* Add check for message key case validity (James D. Forrester)
+* Add option to require a message key prefix (James D. Forrester)
+* build: Replace jshint and jscs with eslint (James Forrester)
```

Comment:
Inconsistent.

Reply:
Aye, that's from git-log. Will fix.

Reply:
Also, today I learned Git doesn't use mailmap for git-log by default, only for git-shortlog. But, it is [configurable](https://stackoverflow.com/a/30900151/319266).